spec/linkedlist.js

```diff
@@ -0,0 +1,238 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import LinkedList from '../src/linkedlist'
+
+chai.use(chaiChange)
+
+describe('LinkedList', () => {
+
+  it('exists', () => {
+    expect(LinkedList).to.be.a('function')
+  })
+
+  context('getHeadNode()', () => {
+    it('Returns the first node in the list', () => {
+      const myLinkedList = new LinkedList()
+      myLinkedList.insert("BMW")
+      myLinkedList.insert("Tesla")
+      myLinkedList.insert("Mercedes")
+      myLinkedList.insertFirst("Lexus")
```

Comment:
your test on line 28 can go right above this line, that way you can test both states in one it